content/browser/ml/webnn/dml/upload_heap.h

```diff
@@ -25,19 +27,15 @@ class UploadHeap final {
   ~UploadHeap();
```

Comment:
Maybe UploadResource is a better name?

Reply:
LGTM, thanks for suggestion, and the ReadBackHeap also will be renamed to ReadbackResource.